tests/Unit/Evolution/Systems/Cce/BoundaryData.py

```diff
@@ -50,9 +50,10 @@ def null_metric(cartesian_to_angular_jacobian, pi, psi):
                                     psi[0, 1:4])[1:3]
```

Comment:
Which formatter are you using? I'm happy to move to whichever, but I had used yapf for this. I just don't want to end up making a bunch of unintentional whitespace changes the next time I come back to these python files.

Reply:
Looks like the whitespace change here is attributable to yapf version changes, from 0.24.0 to 0.29.0

Reply:
I've added a `.style.yapf` file in a commit that should make more versions agree. I'm putting together another PR that will install yapf into the Docker container as well as have GitHub actions check the python files for formatting.

Reply:
is this file already formatted with that `.style.yapf`?

Reply:
Yes